homeassistant/components/plex/helpers.py

```diff
@@ -0,0 +1,25 @@
+"""Helper methods for common Plex integration operations."""
+
+
+def pretty_title(media):
+    """Return a formatted title for the given media item."""
+    year = None
+    if media.type == "album":
+        title = f"{media.parentTitle} - {media.title}"
+    elif media.type == "episode":
+        title = (
+            f"{media.grandparentTitle} - {media.seasonEpisode.upper()} - {media.title}"
```

Comment:
I think that this is not a good idea, long series names will now be indistinguishable because the first 100 chars are all the same.

Reply:
I'd argue this is an issue that's better solved in the UI, and/or by splitting media descriptions into discrete metadata fields.

Reply:
Updated this so the show name is omitted when browsing into the specific show. However, the show name will be printed in directories with mixed shows/media, such as on-deck episodes or recently added items.

Also moved the release year of a TV show from the episode to its season, which should conserve a bit more space.